Volume/Views/AboutUsView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  AboutUsView.swift
+//  Volume
+//
+//  Created by Cameron Russell on 4/12/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct AboutUsView: View {
+    var body: some View {
+        let kvPairs = Constants.subteams.sorted { $0.0.compare($1.0, options: .caseInsensitive) == .orderedAscending }
+        let subteams = kvPairs.map { $0.key }
+        let subteamMembers = kvPairs.map { $0.value }
```

Comment:
It would be simpler to say

```suggestion
        let subteams = kvPairs.keys.sorted()
        let subteamMembers = subteams.map { kvPairs[$0] }
```